ui-modules/blueprint-composer/app/views/main/yaml/yaml.state.js

```diff
@@ -46,6 +46,7 @@ function yamlStateController($scope, $rootScope, $timeout, blueprintService, brS
 
             try {
                 blueprintService.setFromYaml(cm.getValue(), true);
+                blueprintService.refreshBlueprintMetadata();
```

Comment:
There is no point of refreshing the metadata here as the YAML state doesn't use this info. When you flip to the graphical view, this method will be called

Reply:
It could be useful for things in the YAML editor.  Specifically there is a downstream project which wants this for action enablement.  I could also see it wanted for code completion.

Reply:
have commented out and added comments.  ideally we'd have a cache and then re-enable this but agree with you as it stands it is likely to be too chatty with this enabled.